spec/features/card_spec.rb

```diff
@@ -10,13 +10,13 @@
   end
 
   context 'When user try to create card' do
-    it 'not allows to create empty cards' do
+    it 'not allows to create empty cards_mailer' do
       visit_create_card_path
       click_button 'Создать'
       expect(page).to have_content('Заполните все поля!')
     end
 
-    it 'allows to create cards in deck' do
+    it 'allows to create cards_mailer in deck' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.